tests/puppeteer/pages/BO/addProduct

```diff
@@ -0,0 +1,91 @@
+const BOBasePage = require('../BO/BObasePage');
```

Comment:
Pretty sure you forgot to add the `.js` extension in this one too 😝

Reply:
yes, I sure too :smile: 
done thanks